### ISSUE TITLE
[Adaptive Cards]: Links are not discriptive enough to understand as labels are not associate with the links.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7865]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Run the Narrator using "Control+ Windows +Enter" short key in the scan mode.

Repro Steps:
1. Open above URL with V- credential.
2. Run the Narrator using "Control+ Windows +Enter" short key.
3. Navigate to 'Adaptive Cards' link using tab key and press enter key to activate it.
4. Navigate to 'Get Started" & "Watch Video" links using tab key and observe the issue here.

Actual Result:
Narrator is announcing it as "Get Started link" and "Watch Video".

Expected Result:
Narrator should announce "Get Started" and "Watch Video" links as more descriptively like "Adaptive Cards Get started link" and "Adaptive cards Watch Video" link.

Observation:
Same issue present on below paths as well.
1.In the Adaptive Cards main content page-Links present under the "Made For" &"Releases" &"GitHub".
PFA: 3. Label name is missing for the links
2.Click on the "All Microsoft Link" from the header Part-Links present under the "Software", ""business ....etc.
PFA: 3. Labeling name should be given for all the Microsoft links
3."Try it yourself" link present under the all "adaptive cards".
PFA: 3.Try it yourself link label issue
4.Links present under all the card elements from the left navigation pane.
PFA: 3.Card elements label issue

User Experience:
Who are all dependent on-screen reader will not be able to get descriptive information regarding these links if these links are all not descriptive enough.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.05.03

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104482612/190333251-d2c0024f-0703-460b-a362-3ed1f24f370d.mp4

https://user-images.githubusercontent.com/104482612/190333267-24ead6c3-829f-40a5-8683-2fb52fa169aa.mp4

https://user-images.githubusercontent.com/104482612/190333288-28420db3-ba85-4c20-b782-d1e1ec97e28f.mp4

https://user-images.githubusercontent.com/104482612/190341125-4bd2e17c-aeeb-40d2-9ecd-abe92db2d876.mp4

https://user-images.githubusercontent.com/104482612/190341390-83245d6e-5220-4327-b7ec-d34eb324a851.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
Adding A11yTTValidated tags as issue is not repro.
